exec/java-exec/src/main/java/org/apache/drill/exec/physical/config/WindowPOP.java

```diff
@@ -33,26 +35,29 @@
   private final NamedExpression[] withins;
   private final NamedExpression[] aggregations;
   private final Order.Ordering[] orderings;
-  private final long start;
-  private final long end;
+  private final boolean rows;
```

Comment:
The 'rows' variable typically could be misinterpreted to mean number of rows (here you are using it as ROWS vs RANGE)..maybe rename it ?